features/product/viewing_products/viewing_a_products_lowest_price_from_30_days_before_the_discount.feature

```diff
@@ -0,0 +1,76 @@
+@viewing_products
+Feature: Seeing the product's lowest price from 30 days before the discount
+    In order to know whether the current product's price discount is attractive
+    As a Guest
+    I want be be able to see a product's lowest price from 30 days before the discount
+
+    Background:
+        Given the store operates on a single channel in "United States"
+
+    @todo
+    Scenario: Not seeing the lowest price information on a product with no discount
+        Given the store has a product "Wyborowa Vodka" priced at "$21.00"
+        When I check this product's details
+        Then I should not see information about its lowest price
+
+    @todo
+    Scenario: Seeing the lowest price information on a product that has a single discount
+        Given the store has a product "Wyborowa Vodka" priced at "$37.00"
+        And this product's price was discounted to "$21.00"
+        When I check this product's details
+        Then I should see "$21.00" as this product's lowest price from 30 days before the discount
```

Comment:
If I correctly understand the scenario, the lowest price from 30 days before the discount should be the old price => $37.00, as the $21.00 is the current price and it should not be taken into account in the calculation.

Reply:
I was not aware of that behavior